@xen-orchestra/web/src/utils/xo-api-definition.util.ts

```diff
@@ -36,4 +38,16 @@ export const xoApiDefinition = {
     fields: '*',
     handler: (record: XoDashboard) => record,
   },
+  pif: {
+    type: 'collection',
+    path: 'pifs',
+    fields: '$host,$network,attached,carrier,device,dns,gateway,id,ipv6,mac,mode,mtu,netmask,speed',
```

Comment:
In `pif.type.ts` I see a field `ip` that is used in `network.vue` and `PifPanel.vue`. However, it seems to be missing here:
```suggestion
    fields: '$host,$network,attached,carrier,device,dns,gateway,id,ip,ipv6,mac,mode,mtu,netmask,speed',
```